tests/periph_spi/tests/periph_spi.keywords.txt

```diff
@@ -0,0 +1,26 @@
+*** Settings ***
+Library             SPIdevice  port=%{PORT}  baudrate=%{BAUD}  timeout=${%{HIL_CMD_TIMEOUT}}  connect_wait=${%{HIL_CONNECT_WAIT}}  parser=json
+
+Resource            api_shell.keywords.txt
+Resource            philip.keywords.txt
+
+*** Keywords ***
+SPI Init Should Succeed
+    [Documentation]             Fully initialized the SPI
+    API Call Should Succeed     Spi init        %{HIL_SPI_DEV}
```

Comment:
I wouldn't use env variable here, bc now we can only test one SPI device at once, right?

Reply:
So should we have it in every call during the test?  This would also mean we would have to expose the nss port and pin.  Can you think of a test that we would want more than 1 SPI? If so than I would expose this and have a keyword for each call such as:
```
SPI Init Should Succeed Default
      SPI Init Should Succeed %{HIL_SPI_DEV}
```
If you say so I will but I think it is a lot of bulk for something that may not be used.